foxy/distribution.yaml

```diff
@@ -2390,6 +2390,32 @@ repositories:
       url: https://github.com/ros-industrial/ros_canopen.git
       version: dashing-devel
     status: developed
+  ros_cb:
+    doc:
+      type: git
+      url: https://github.com/jediofgever/ROS_CB.git
+      version: master
+    release:
+      packages:
+      - chiconybot
+      - chiconybot_bringup
+      - chiconybot_cartographer
+      - chiconybot_description
+      - chiconybot_gazebo
+      - chiconybot_msgs
+      - chiconybot_navigation2
+      - chiconybot_node
+      - chiconybot_teleop
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/jediofgever/ROS_CB-release.git
+      version: 0.0.0-2
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/jediofgever/ROS_CB-release.git
```

Comment:
Shouldn't this reference the non-release repo?

Reply:
thanks @dirk-thomas . I changed it non-release repo